lib/improver/tests/nowcasting/lightning/test_NowcastLightning.py

```diff
@@ -248,18 +248,6 @@ def test_missing_lightning(self):
                                             self.precip_cube,
                                             None)
 
-    def test_missing_first_guess(self):
```

Comment:
Could we keep this test but check it raises the new error from the new function instead?

Reply:
Re-added test and amended message that is being checked for.